Formula/terraform.rb

```diff
@@ -52,21 +54,27 @@ def install
       buildpath.install "stringer"
     end
 
+    cd "src/github.com/kisielk/errcheck" do
+      system "go", "build"
+      buildpath.install "errcheck"
+    end
+
     cd terrapath do
       # v0.6.12 - source contains tests which fail if these environment variables are set locally.
       ENV.delete "AWS_ACCESS_KEY"
       ENV.delete "AWS_SECRET_KEY"
 
-      terraform_files = `go list ./...`.lines.map { |f| f.strip unless f.include? "/vendor/" }.compact
-      system "go", "test", *terraform_files
+      # Runs format check and test suite via makefile
+      ENV.deparallelize { system "make", "test", "vet" }
```

Comment:
I've spent some time looking at this today, and I'm still not sure what exactly is breaking. I've tried multiple times on my local system with the `-j4` MAKEFILES env variable, similar to the Jenkins environment, and it works every time. So I must be missing something about the way @BrewTestBot is building this, as it's hitting the same race condition every single time that my local machine isn't when attempting to build the bottle.

I don't mind reporting this issue, but I don't feel I have enough diagnostic details to.